### ISSUE TITLE
Arial Font characters getting overlapped on PDF

### ISSUE THREAD

**Issue by VinayParasrampuria** [wkhtmltopdf/wkhtmltopdf#3147]
We are  using wkhtmltopdf version 0.12.4.
I am getting issue with character spacing while using  Arial font. PFA attached samples.

[testArial.pdf](https://github.com/wkhtmltopdf/wkhtmltopdf/files/501027/testArial.pdf)
[testArial.txt](https://github.com/wkhtmltopdf/wkhtmltopdf/files/501033/testArial.txt)

PFB HTMLcode
```html
<!DOCTYPE HTML>

<!DOCTYPE html PUBLIC "" ""><HTML 
xmlns="http://www.w3.org/1999/xhtml"><HEAD><META content="IE=11.0000" 
http-equiv="X-UA-Compatible">

<META http-equiv="X-UA-Compatible" content="IE=edge">  
<META http-equiv="Content-Type" content="text/html; charset=utf-8">
 <TITLE></TITLE>

<Style>
    body {
    margin: 0px; padding: 0px; font-family: Arial; background-color: white ; font-style: normal;
}


.table table {
    width: 80%; border-collapse: collapse; border-spacing: 0;
}
.table th {
    text-align: left; font-size: 10px; font-weight: bold;
}
.table tbody > tr > td {
    border-width: 1px 0px; border-style: solid none; border-color: currentColor; padding-top: 4px; font-size: 12px; font-weight: 100;
}
.highlight-header thead {
    border-top-color: currentColor; border-top-width: 1px; border-top-style: solid; background-color: rgb(197, 234, 239);
}
.table .sub-heading {
    height: 30px; font-weight: bold; vertical-align: bottom;
}
.table td {
    text-align: left;
}
.table th {
    text-align: left;
}
.table td + td {
    text-align: right;
}
.table th + th {
    text-align: right;
}
</Style>


<BODY>

<DIV class="table">
<TABLE>
  <COLGROUP>
  <COL style="width: 26%;" span="1">
  <COL style="width: 25%;" span="1">
  <COL style="width: 25%;" span="1">
  <COL style="width: 23%;" span="1"></COLGROUP>
  <THEAD>
  <TR>
    <TH>J</TH>
    <TH>P</TH>
    <TH>C</TH>
    <TH>C</TH></TR></THEAD>
  <TBODY> 

    <TD>ICOMM Approved Amount</TD>
    <TD></TD>
    <TD></TD>
    <TD>-</TD></TR>
  <TR>
    <TD>Gross IRR Realized to Date</TD>
    <TD></TD>
    <TD></TD>
    <TD></TD></TR>
  <TR>
    <TD>Gross CFx Realized to Date</TD>
    <TD></TD>
    <TD></TD>
    <TD></TD></TR>
  <TR>
    <TD>Total Projected IRR</TD>
    <TD></TD>
    <TD></TD>
    <TD></TD></TR>
  </TBODY></TABLE></DIV>


 </BODY></HTML>
```

Regards
Vinay

**Comment by VinayParasrampuria** [comment]
It seems Issue is only when the font size is between 10.5px to 11.4px . I tried other then this range and its working good.

Regards
Vinay

**Comment by migacz1125** [comment]
I confirm this issue if i use Arial font. If i have been used other font like Verdana issue doesn't exist

**Comment by GentlemanK** [comment]
I am also having issues with the ARIAL fonts for sizes less than 12. 

![image](https://user-images.githubusercontent.com/27438421/28732480-07bbb43a-73a6-11e7-8059-ebcb868a9e05.png)

Any idea why?

**Comment by nrg294** [comment]
this issue has not been commented on for a while any update?

**Comment by Tomsgu** [comment]
Closing as the original issue was fixed, because I can't reproduce it in 0.12.5. Please let me know if it persists.

@GentlemanK your issue is already posted in #1524.
